lola.html

```diff
@@ -0,0 +1,953 @@
+        <li>Private items are in scope if the user authorizes</li>
+        <li>Optionally, if the user asks later, the source server can notify follows of the source account that the account has moved and what 3rd party recipients can do with these notifications</li>
+        <li>Optionally, if the user asks later, the source server can redirect requests for content with URLs for the destination server.</li>
+      </ul>
+
+    </section>
+
+    <section class="informative">
+      <h2>Use cases</h2>
+
+
+      <p>The primary user need to solve with LOLA is to allow Fediverse users to move their accounts in response to moderation and defederation decisions.  The connections between servers and the communities on servers form a structure that has close connections, looser connections and blocks, allowing many different types of communities to co-exist.  That does put a lot of pressure on users to be able to move their accounts (due to moderation decisions they disagree with, because of server defederation, or just to move closer to an affinity group).   A user making such an account move will sometimes want to copy their previously posted content to their new location, sometimes follow the same people, and sometimes notify their followers so their followers can resubscribe in a new location.  Servers implementing LOLA can provide these choices to users.</p>
+
+      <p>Several secondary use cases are enabled by this approach.  It allows accounts to be copied and NOT moved which is useful for testing of all kinds. The collections defined with migration data can lead smoothly to an interoperable export format that can share much code with migration support.  The mechanisms are independent of each other (beyond assuming the account migration authorization token), thus allowing reuse - e.g. the mechanisms for notify/redirect can also be used after a user moves to a new location using a different approach.  <p>
+
+      <p>Some use cases are not covered by this approach.  An offline server cannot participate in server-to-server portability mechanisms.  Servers may also be unwilling or unable to communicate directly in any way.  If the unwillingness to communicate is part of ‘defederating’ another server, we recommend that account portability be an exception to the defederation logic, as moving accounts is an important part of helping defederation work well for more people.</p>
+
+      <p>The <a href="https://codeberg.org/fediverse/fep/src/branch/main/fep/73cd/fep-73cd.md">Migration User Stories</a> FEP has a number of use cases that can be solved with LOLA, as well as some that can't.  Those intended to be addressed: </p>
+
+      <ul>
+        <li>1A Account-level redirect: Yes</li>
+        <li>1B content-level redirect to a home page: Yes if source can manage it</li>
+        <li>1C Content-level redirect to same content: Possibly - needs more discussion</li>
```

Comment:
```suggestion
        <li>1C Activity-level redirect to same content: Possibly - needs more discussion</li>
```

I would also mention that this might be better specified in an independent FEP on another timeline, or specified in a FEP that stays in "draft" status while export format gets implemented, in case corner cases arise from the interplay of a LOLA approach and an export/import approach?